### PR TITLE
[HOUSEKEEPING] have error identifiers for two errors in azure git lib

### PR DESCRIPTION
**Pull request by dennisseah**

### CODE REVIEW

**Review by yradsmikham — APPROVED** [review]

**Comment by yradsmikham** [comment]
Is there is an issue associated with this?

**Comment by dennisseah** [comment]
> Is there is an issue associated with this?

@yradsmikham, @andrebriggs and @evanlouie are ok with adding a card in https://github.com/microsoft/bedrock/projects/42 for small housekeeping tasks like this one.

File: src/lib/git/azure.ts [hunk 1]
```diff
@@ -366,11 +366,10 @@ export const repositoryHasFile = async (
   );
 
   if (gitItem === null) {
-    throw Error(
-      "Error installing build pipeline. Repository does not have a " +
-        fileName +
-        " file."
-    );
+    throw buildError(errorStatusCode.GIT_OPS_ERR, {
+      errorKey: "git-azure-file-no-exist-in-repo",
```
Comment (bnookala) [inline src/lib/git/azure.ts:370]: can we just call this git-azure-file-exist? i don't see a reason why we'd throw an error _if_ the file exists

File: src/lib/git/azure.ts [hunk 2]
```diff
@@ -393,9 +392,10 @@ export const validateRepository = async (
   const repo = await gitApi.getRepository(repoName, project);
 
   if (!repo) {
-    throw Error(
-      `Project '${project}' does not contain repository '${repoName}'.`
-    );
+    throw buildError(errorStatusCode.GIT_OPS_ERR, {
+      errorKey: "git-azure-repo-no-exist",
```
Comment (bnookala) [inline src/lib/git/azure.ts:396]: same feedback here

**Review by bnookala — CHANGES_REQUESTED** [review]

File: src/lib/git/azure.ts (hunk 1 above)
Comment (dennisseah) [reply src/lib/git/azure.ts:370]:
```
  if (gitItem === null) {
    throw buildError(errorStatusCode.GIT_OPS_ERR, {
      errorKey: "git-azure-file-no-exist-in-repo",
      values: [fileName, branch, repoName],
    });
  }
```
if `gititem === null` means that file does not exist. Am I right? Or it is the other way around. `gititem === null` means file exist?

**Review by bnookala — APPROVED** [review]

**Pull request merged**